### PR TITLE
Obsdocs 1360 - 1x.pico sizing for loki

### PR DESCRIPTION
**Pull request by libander**
Version(s): OCP 4.15, 4.16, 4.17, 4.18

Issue: [OBSDOCS-1360](https://issues.redhat.com//browse/OBSDOCS-1360)

Link to docs preview: https://84655--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging-6.0/log6x-loki.html

QE review: 
- [x] QE has approved this change.

### CODE REVIEW

**Comment by libander** [comment]
@periklis / @xperimental - Review please. 
@anpingli - Review please.

**Comment by hideshi3** [comment]
The number of columns in the table "Loki sizing" (in https://84655--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging-6.0/log6x-loki.html) is incorrect, so the contents of the table are broken.

File: modules/log6x-configuring-otlp-output.adoc
Comment (lpettyjo) [inline modules/log6x-configuring-otlp-output.adoc:50]: Not wrong to indicate TP status here, but you already have the TP notice above, so is this really necessary? It might not get changed when the feature transitions to GA. Your call though.

**Review by lpettyjo — COMMENTED** [review]
You have two commits that need to be squashed to one. For information about how to do this, see the OpenShift Docs Manual "Squash multiple commits into one commit" section (https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit?tab=t.0#heading=h.epes4xf06pzk). Let me know if you have any questions.

Otherwise, LGTM!

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:19]: 1x.demo  is for test only. Please specify this point. That is very important.

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:48]: It is 6.8 from my observation.  as request resource isn't specified for opa.    I think 7. is the correct value.

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:55]: It is 7.8 from my observation.  as request resource isn't specified for opa.    I think 8. is the correct value.

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:62]: It is  17Gi from my observation. 6.9Gi + 10Gi  =~ 17Gi

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:70]: It is  18Gi from my observation. 6.9Gi + 11Gi  =~ 18Gi

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [reply modules/log6x-loki-sizing.adoc:48]: @xperimental  @periklis  can you confirm if this is as expected.

Comment (xperimental) [reply modules/log6x-loki-sizing.adoc:48]: Yes, that is correct. Pico requests 6.8 vCPU without and 7.8 with the ruler.

File: modules/log6x-loki-sizing.adoc
Comment (xperimental) [reply modules/log6x-loki-sizing.adoc:62]: I count 16288 MiB, which is just below 16GiB. With the ruler it will be 1GiB more.

File: modules/log6x-loki-sizing.adoc
Comment (xperimental) [inline modules/log6x-loki-sizing.adoc:77]: I'm getting 590 GiB without and 910 GiB with the ruler.
I think we forgot to adjust the ingester's PVC sizes when we added the third ingester.

For now, let's go with the sizes that will be in the release, we probably need to adjust this down in a z-stream.

File: modules/log6x-configuring-otlp-output.adoc
Comment (libander) [reply modules/log6x-configuring-otlp-output.adoc:50]: Eh, this file isn't my commit, so I'll leave that up to the author. Good eyes, though!

File: modules/log6x-loki-sizing.adoc
Comment (libander) [reply modules/log6x-loki-sizing.adoc:62]: I'm going with 17 because it seems better to overestimate than underestimate in terms of customer capacity planning, any objections?

File: modules/log6x-loki-sizing.adoc
Comment (libander) [reply modules/log6x-loki-sizing.adoc:77]: @xperimental - it seems a bit off that the disk request for pico is more than the larger sizes, I would expect that to potentially cause confusions - is there there a callout we can add to mitigate?

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:48]: how about use 7 here?

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [inline modules/log6x-loki-sizing.adoc:55]: How about use 8 here. That is a approximate value. I think it is better to a little bigger than the actual value.

File: modules/log6x-loki-sizing.adoc
Comment (anpingli) [reply modules/log6x-loki-sizing.adoc:48]: That is a approximate value. I think it is better to be a little bigger than the actual value.

**Comment by libander** [comment]
@anpingli - lgtm please.

File: observability/logging/logging-6.0/log6x-clf.adoc
Comment (xperimental) [inline observability/logging/logging-6.0/log6x-clf.adoc:91]: Was this line and the "configuring otlp output" file added to this PR intentionally? It seems unrelated to the pico size and the release-notes PR also seems to include the same file.

**Comment by libander** [comment]
@xperimental - lgtm please

File: modules/log6x-loki-sizing.adoc
Comment (xperimental) [inline modules/log6x-loki-sizing.adoc:12]: Do we want to mention that `1x.pico` is new here again instead of just on the release notes and in the table below?

I think the second paragraph comes out of the short discussion we had in the arch meeting :+1: 

```suggestion
Disk requests are similar across size configurations, allowing customers to test different sizes to determine the best fit for their deployment needs.
```

Comment (libander) [reply modules/log6x-loki-sizing.adoc:12]: I think the 6.1+ covers that. And you are correct, thanks for explaining!

**Comment by anpingli** [comment]
LGTM

File: modules/log6x-loki-sizing.adoc
Comment (briandooley) [reply modules/log6x-loki-sizing.adoc:12]: This will be returned to in https://issues.redhat.com/browse/OBSDOCS-1494

**Review by xperimental — APPROVED** [review]

File: modules/log6x-loki-sizing.adoc
Comment (mburke5678) [inline modules/log6x-loki-sizing.adoc:8]:
```suggestion
Sizing for Loki follows the `1x.<size>`format, where `1x` is number of instances and `<size>` specifies performance capabilities.
```

File: modules/log6x-loki-sizing.adoc
Comment (mburke5678) [inline modules/log6x-loki-sizing.adoc:10]:
```suggestion
For logging 6.1+, the `1x.pico` configuration defines a single Loki deployment with minimal resource and limit requirements, offering high availability (HA) support for all Loki components. This configuration is suited for deployments that do not require a single replication factor or auto-compaction.
```

Comment (mburke5678) [reply modules/log6x-loki-sizing.adoc:10]: If this paragraph does not apply to logging 6.0, I don't think it should be in the 6.0 docs, right?

**Comment by mburke5678** [comment]
@libander A few suggestions. Otherwise LGTM

File: modules/log6x-loki-sizing.adoc
Comment (libander) [reply modules/log6x-loki-sizing.adoc:8]: The current wording has been in the docs for awhile, I see no reason to make this instance variant.

File: modules/log6x-loki-sizing.adoc
Comment (libander) [reply modules/log6x-loki-sizing.adoc:10]: This is into a release branch, so it'll get scooted over to the correct area before merging to main. :)

**Review by skrthomas — COMMENTED** [review]
merge review lgtm

**Comment by skrthomas** [comment]
Had a moment of panic that I didn't cherrypick these but forgot that we used a release branch and the cherrypicks were there. 😮‍💨

**Pull request merged**
